samples/web/content/apprtc/js/appcontroller.js

```diff
@@ -288,10 +341,17 @@ AppController.prototype.onKeyPress_ = function(event) {
   }
 };
 
-AppController.prototype.displaySharingInfo_ = function(roomLink) {
+AppController.prototype.pushCallNavigation_ = function(roomId, roomLink) {
+  if (!isChromeApp()) {
+    window.history.pushState({'roomId': roomId, 'roomLink': roomLink }, roomId, roomLink);
+  }  
+};
+
+AppController.prototype.displaySharingInfo_ = function(roomId, roomLink) {
   this.roomLinkHref_.href = roomLink;
   this.roomLinkHref_.text = roomLink;
   this.roomLink_ = roomLink;
+  this.pushCallNavigation_(roomId, roomLink);
```

Comment:
displaySharingInfo_ shouldn't be responsible for this. Not sure why we need it actually..